React/main/src/app/components/Topbar/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import { NavLink, withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import authActions from '~/../redux/auth/actions';
+
+import style from './styles.scss';
+
+class Topbar extends Component {
+  state = {
+    menuIsCollapsed: true
+  };
+
+  componentDidMount() {
+    window.addEventListener('click', this.handleClickEvent);
+  }
+
+  toggleMenu = () => {
+    this.setState({ menuIsCollapsed: !this.state.menuIsCollapsed });
```

Comment:
Change it to this.setState(prevState => ({ menuIsCollapsed: !prevState.menuIsCollpased }))